src/keys/.gitignore

```diff
@@ -1,2 +0,0 @@
-*
-!.gitignore
```

Comment:
Since we're removing the _keys/_ folder from the deployment, please update the _bin/deploy_ script as follows:

* Change line 17 to `-not -name 'key.dat'`.
* Remove line 18.

If you could test that it works correctly on a copy of the current deployment (i.e. the existing _keys/_ folder gets deleted and the existing _key.dat_ file is left intact), that would be great.